src/Search/Search.svelte

```diff
@@ -79,6 +79,7 @@
   $: dispatch(expanded ? "expand" : "collapse");
 </script>
 
+<!-- svelte-ignore a11y-autofocus -->
```

Comment:
Long-term, I recommend we remove `autofocus` functionality and guide users to instead utilize `ref` to manually set focus based on user or server events.

Reply:
Agreed – would love to sort out the ignorable a11y warnings from the rest.